k8s/cse/import-corner/d2cc/data-bridge/cse-import-d2cc-automated-forced-pras-data-bridge-deployment.yaml

```diff
@@ -0,0 +1,110 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: cse-import-d2cc-automated-forced-pras-data-bridge
+  labels:
+    app: gridcapa-data-bridge
+    component: gridcapa-automated-forced-pras-data-bridge
+    region: cse
+    process: import-d2cc
+
+spec:
+  selector:
+    matchLabels:
+      app: gridcapa-data-bridge
+      component: gridcapa-automated-forced-pras-data-bridge
+      region: cse
+      process: import-d2cc
+  template:
+    metadata:
+      labels:
+        app: gridcapa-data-bridge
+        component: gridcapa-automated-forced-pras-data-bridge
+        region: cse
+        process: import-d2cc
+    spec:
+      containers:
+        - name: cse-import-d2cc-automated-forced-pras-data-bridge
+          image: gridcapa-cse-input-data-bridge
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+          env:
+            - name: SOURCE_HOST
+              valueFrom:
+                configMapKeyRef:
+                  name: env-configmap
+                  key: GRIDCAPA_SOURCE_HOST_CSE_IMPORT_D2CC
+            - name: SOURCE_PORT
+              valueFrom:
+                configMapKeyRef:
+                  name: env-configmap
+                  key: GRIDCAPA_SOURCE_PORT_CSE_IMPORT_D2CC
+            - name: SOURCE_USER
+              valueFrom:
+                secretKeyRef:
+                  name: gridcapa-ftp-credentials
+                  key: ftp-user
+            - name: SOURCE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: gridcapa-ftp-credentials
+                  key: ftp-password
+            - name: SOURCE_BASE_DIRECTORY
+              valueFrom:
+                configMapKeyRef:
+                  name: env-configmap
+                  key: GRIDCAPA_SOURCE_BASE_DIRECTORY_CSE_IMPORT_D2CC
+            - name: MINIO_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: gridcapa-minio-credentials
+                  key: minio-access-key
+            - name: MINIO_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: gridcapa-minio-credentials
+                  key: minio-secret-key
+            - name: MINIO_BUCKET
+              valueFrom:
+                configMapKeyRef:
+                  name: env-configmap
+                  key: GRIDCAPA_MINIO_BUCKET_CSE_IMPORT_D2CC
+            - name: MINIO_BASE_DIRECTORY
+              valueFrom:
+                configMapKeyRef:
+                  name: env-configmap
+                  key: GRIDCAPA_MINIO_BASE_DIRECTORY_CSE_IMPORT_D2CC
+          volumeMounts:
+            - mountPath: /config/application.yml
+              subPath: application.yml
+              name: cse-import-d2cc-automated-forced-pras-data-bridge-configmap-volume
```

Comment:
Name too long to be handled by k8s you can rename `cse-import-d2cc-automated-forced-pras-data-bridge`